enhancements/microshift/microshift-default-cni.md

```diff
@@ -0,0 +1,290 @@
+---
+title: microshift-default-cni
+authors:
+  - majopela
+  - zshi-redhat
+  - dhellmann
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@sallyom, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@dcbw, OpenShift Networking Manager"
+approvers:
+  - "@derekwaynecarr"
+api-approvers:
+  - None
+creation-date: 2022-07-18
+last-updated: 2022-07-18
+tracking-link:
+  - https://issues.redhat.com/browse/USHIFT-40
+see-also:
+  - https://github.com/openshift/microshift/blob/main/docs/design.md
+  - https://github.com/openshift/enhancements/pull/1187
+---
+
+# Default CNI 
+
+## Summary
+
+This enhancement proposes the adoption of a default MicroShift CNI.
+
+MicroShift addresses customer use cases with low-resource,
+field-deployed edge devices (SBCs, SoCs) requiring a minimal K8s
+container orchestration layer.
+
+MicroShift is targeting a class of devices like Xilinx ZYNQ
+UltraScale+, fitlet2, NVIDIA Jetson TX2, or Intel NUCs that are
+pervasive in edge deployments. These cost a few hundred USD, have all
+necessary accelerators and I/O on-board, but are typically not
+extensible and are highly constrained on every resource, e.g.:
+
+* CPU: ARM Cortex-A35-class or Intel Atom class CPU, 2-4 cores, 1.5GHz clock
+* memory: 2-16GB RAM
+* storage: e.g. SATA3 @ 6Gb/s, 10kIOPS, 1ms (vs. NVMe @ 32Gb/s,
+  500kIOPS, 10µs)
+* network: Less bandwidth and less reliable than data center-based
+  servers, including being completely disconnected for extended
+  periods. Likely 1Gb/s NICs, potentially even LTE/4G connections at
+  5-10Mbps, instead of 10/25/40Gb/s NICs
+
+
+## Motivation
```

Comment:
I miss a paragraph on the motivation section indicating why bridge-cni (despite being the simplest) is not enough for what you want to achieve.

As I understand, everything in this section aims at minimal resource consumption - and only that. Something which, is granted by bridge-cni. Hence, it is not clear to me (from this paragraph) what motivates you to have ovn-k as the default cni for microshift.